site/src/utilities.ts

```diff
@@ -386,3 +386,46 @@ export function trackView(url: string, referrer?: string): void {
     console.error(e);
   }
 }
+
+export function getLogo(): string {
+  const baseUrl = `${shortSemToURL()(process.env.VUE_APP_CURR_SEM)}/img/logos/`;
+  const defaultLogo = ["quacs_logo.svg", "quacs_logo_white_duck.svg"];
```

Comment:
What's the difference between these two logos?

Reply:
quacs_logo is what we use now and quacs_logo_white was just lying around. The only difference seems to be that quacs_logo_white has the question mark filled in as white instead of transparent.

I was mostly using this to verify behaviors of having multiple logos, I can remove whatever from this